apps/src/sites/studio/pages/jotformLoader.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * @file Entry point for a bundle to embed on pages where we are also letting JotForm embed
+ * an inline form using an iframe.  This bundle monitors the JotForm script and attempts
+ * to measure reachability of the JotForm domain, load success, load time, and reports metrics
+ * back to our system so we have a sense of how often this fails.
+ * We also attempt to improve the user experience when the form is unavailable.
+ */
+import logToCloud from '../../../logToCloud';
+
+function main(context) {
+  Promise.all([
+    checkJotFormFrameLoaded(context),
+    // Domains used by Jotform:
+    // cdn.jotfor.ms       https://cdn.jotfor.ms/images/calendar.png  817B
```

Comment:
`817B` is the file size?

Reply:
Yep! Just wanted to document that we found a small file to check on the target domain.